packages/contracts-bedrock/test/L1/DataAvailabilityChallenge.t.sol

```diff
@@ -0,0 +1,504 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    DataAvailabilityChallenge,
+    ChallengeStatus,
+    Challenge,
+    CommitmentType,
+    computeCommitmentKeccak256
+} from "src/L1/DataAvailabilityChallenge.sol";
+import { Proxy } from "src/universal/Proxy.sol";
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+contract DataAvailabilityChallengeTest is CommonTest {
+    DataAvailabilityChallenge public dac;
+
+    function setUp() public virtual override {
+        super.enablePlasma();
+        super.setUp();
+        dac = DataAvailabilityChallenge(deploy.mustGetAddress("DataAvailabilityChallengeProxy"));
```

Comment:
Inside of `Setup.sol` you create a variable `dataAvailabilityChallenge` so you don't need to make `dac` here?

Reply:
done